src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Dispatch } from 'react';
+
+interface TabsInterface {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: TabsInterface[]
+  selectedTab: TabsInterface
+  onSelect: Dispatch<TabsInterface>
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  onSelect,
+}) => (
+  <div>
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={
+              cn('', '', {
+                'is-active': tab === selectedTab,
+              })
+            }
```

Comment:
```suggestion
              cn({
                'is-active': tab === selectedTab,
              })
            }
```
why do you need these empty classnames?